.env.example

```diff
@@ -367,3 +367,6 @@ RETELL_AI_KEY=
 
 # Used to disallow emails as being added as guests on bookings
 BLACKLISTED_GUEST_EMAILS=
+
+# Used to send alert email when daily webhooks goes down
+ALERT_EMAIL_TO=
```

Comment:
If it is specific to daily webhook, we should be clear about it in the name.

Reply:
i think we can use this variable in the future for any other thing to send trigger notification to use

Reply:
Ok for now i have changed the name `DAILY_WEBHOOKS_ALERT_EMAIL_TO`